src/supermodular.jl

```diff
@@ -0,0 +1,7 @@
+# Katrina Evtimova, kve216
+
+function is_supermodular(player::Player{2})
```

Comment:
I'm not sure individual functions justify being in their own file? Are you planning on adding more functions to this file? Otherwise it may make sense to just put this in some sort of utils file